nixos/modules/services/misc/gitlab.nix

```diff
@@ -560,6 +560,7 @@ in {
         mkdir -p ${cfg.statePath}/tmp/sockets
         mkdir -p ${cfg.statePath}/shell
         mkdir -p ${cfg.statePath}/db
+        mkdir -p ${cfg.statePath}/uploads
```

Comment:
Are you sure that this is necessary, as far as I had understood `uploads.storage_path = cfg.statePath;` in the config above should have made it possible yo remove this. How did you trigger an error?